corehq/apps/cloudcare/views.py

```diff
@@ -126,31 +126,38 @@ def get_restore_as_user(request, domain):
         if not hasattr(request, 'couch_user'):
             raise Http404()
 
-        def set_cookie(response):  # set_coookie is a noop by default
+        def set_cookie(response):  # set_cookie is a noop by default
             return response
 
+        def _get_login_as_user_query():
+            return login_as_user_query(
+                domain,
+                request.couch_user,
+                search_string='',
+                limit=500,
```

Comment:
Should this go back to 1 now?
```suggestion
                limit=1,
```